### DISCUSSION TITLE
Concerns about data protection on the homepage

### DISCUSSION THREAD

**Discussion by linux-lukas** [VSCodium/vscodium#1014] [General]
Personally, I am a very big friend of VSCodium and use it (almost) daily. 

But in the last few days I have noticed a few things about the project homepage (vscodium.com) that I find somewhat paradoxical.

1. why are contents from Google (googleusercontent.com and fonts.googleapis.com, bootstrap [external]) and also from Microsoft (azure.com) used on the website?

2. why is the homepage hosted on an Amazon server (in the form of Cloudflare, which belongs to Amazon)? 

3. To what extent are the integrations of content from the domains shields.io and travis-ci.com necessary? 

As I mentioned at the beginning, I am a great friend of this open source project, but it leaves a strange aftertaste. :/

**Comment by paulcarroty** [comment]
1. 
* googleusercontent.com - not found
* fonts.googleapis.com - google fonts CDN, can be replaced to local fonts.
* azure.com - VSCodium CI status 

2. Hosted on Cloudflare CDN, not related to Amazon at all.

3.
* shields.io - for shields (surprisingly!) line current release, chat icon etc.
* travis-ci.com - used for CI status in the past, now can be removed.

If you wanna to improve something - just sent your PRs.

**Reply by linux-lukas** [reply]
Thank you very much for your detailed answer! 

1. Could the procedure that is handled via "azure.com" be replaced by a more privacy-friendly & functionally functional open source solution? 

2. Could the Cloudflare CDN also be replaced by a more privacy-friendly & functionally functional open source solution?

**Reply by paulcarroty** [reply]
Yes. 1st now fixed, 2nd - hard to believe in open source CF alternative.

**Reply by daiyam** [reply]
> * azure.com - VSCodium CI status

It can also be removed since it was used to build windows releases until Jan 13, 2021 (https://github.com/VSCodium/vscodium/commit/9dd4e545c0c42c6e5001393b28768f1c4c1e787c).

Now, Github Actions is the only used CI.

**Reply by paulcarroty** [reply]
Fixed.

**Comment by linux-lukas** [comment]
At https://european-alternatives.eu/alternative-to/cloudflare, the following Cloudflare alternatives are mentioned, among others: 

- keycdn (project page: https://www.keycdn.com/ ; GitHub: https://github.com/keycdn); Switzerland
- OVHcloud CDN (https://www.ovhcloud.com/en/web-hosting/options/cdn/ , documentation for using the OVHcloud CDN for the website: https://docs.ovh.com/us/en/hosting/guide_to_using_the_geocache_accelerator_on_a_web_hosting_package/) ; France / Germany

Perhaps in the medium term it would really make sense to look through the list of Cloudflare alternatives mentioned in the first link, compare it with Cloudflare and then plan which preparatory steps need to be taken next so that in the long term it is possible to switch to an alternative with equivalent or even better workflows and everything else that goes with it. Because with a move away from Cloudflare to another CDN provider, e.g. from the EU, legal problems become less (due to binding regulations from GDPR).
